planning/Discovery/work-items.md

```diff
@@ -21,40 +21,52 @@ Prioritized list of work items related to Discovery.
-10. Dependency - [Interoperability Profiles](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#profiles-workitem)
+9. Dependency - [Interoperability Profiles](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#profiles-workitem)
     * Should TDDs also satisfy a Profile?
-11. Process - [WoT CG Coordination](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#wotcg-coordination)
+10. Process - [WoT CG Coordination](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#wotcg-coordination)
     * Gathering information on requirements, implementations, feedback on current versions, suggestions for improvement, etc.
-12. Direct - Placeholder - Sort-by Filtering
-13. Direct - Placeholder - Validation
-    * Subset of [Discovery Improvements](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-improvements-workitem)
-14. Direct - Placeholder - TD Versions
+11. Direct - Placeholder - Validation
     * Subset of [Discovery Improvements](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-improvements-workitem)
-15. Direct - Placeholder - Version Selection
+12. Direct - Placeholder - TD Versions
     * Subset of [Discovery Improvements](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-improvements-workitem)
-16. Direct - Placeholder - Improve Security
+13. Direct - Placeholder - Version Selection
     * Subset of [Discovery Improvements](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-improvements-workitem)
-17. Dependency, Tentative - [Linting mechanism for TDs](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#td-linting-workitem)
-    * Tentative: if meant for development may not be relevant to TDDs; to discuss.
-18. Direct, Tentative - [Discovery of Thing Models](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-thing-models-workitem)
-    * Tentative - needs use cases clarified, we may or may not do it.
-19. Direct - Placeholder - Additional Introductions
+14. Direct - Placeholder - Additional Introductions
     * To be researched, but to support additional ecosystems
     * Examples: MQTT Discovery, OPC UA Discovery
+    
+#### Middle Priority
+1. Direct - [Discovery CoAP Directory](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#discovery-coap-dir-workitem)
+2. Dependency - [Geolocation](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#geolocation-workitem)
+    * Assuming geolocation data added to TDs, this would add geolocation query support to Discovery.
+    * May be higher priority for some use cases - more analysis needed
+    * Requirements: [https://github.com/w3c/wot-architecture/blob/main/REQUIREMENTS/geolocation.md]
+3. Dependency - [Onboarding](https://w3c.github.io/wot-charter-drafts/wot-wg-2023-details.html#onboarding-workitem)
```

Comment:
Do we want to provide few optional onboarding/pairing patterns? It could be in itself a task as big as the whole `Directory` effort. 

I'd move it to lower priority if it isn't felt necessary for one of the big use-case for Directory/Discovery.

Reply:
I personally don't think "Onboarding" should be discussed as part of Discovery, it's a separate topic.  For now it has been moved down to a "Questionable" section.